data/crd/ssp.kubevirt.io_ssps.yaml

```diff
@@ -0,0 +1,862 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This file is not related to this PR. It was generated, because we forgot to add it to `.gitignore`.